### PR TITLE
Feedback on Hardware pages

### PR DESCRIPTION
**Pull request by raphaelcherney**
Still want to add more visuals in a few places, but this is a start.

### CODE REVIEW

File: docs/hw/mechanical.md
Comment (alsora) [inline docs/hw/mechanical.md:24]: nice!
this renders much more nicely than the footers notes that we have in the electrical and hookup examples pages

**Review by alsora — APPROVED** [review]

File: docs/hw/electrical.md
Comment (shamlian) [inline docs/hw/electrical.md:34]: I disagree with this becoming an info box. The information is extremely important and should be under a subheading. (By doing this, you have put it under the table of contents under the USB C connector, which doesn't make sense.) I have found with Create 2 that people are generally confused about what volts, amps, and watts are, and how they are used. At the very least, I would suggest the title and first sentence be formatted as heading and text; the rest can be in an info box.

File: docs/hw/mechanical.md
Comment (shamlian) [inline docs/hw/mechanical.md:2]: I feel like it's a little visually jarring to start this page with two warning boxes. Is there any explanatory text we could put before them (like on the electrical page)?

File: docs/hw/mechanical.md
Comment (shamlian) [inline docs/hw/mechanical.md:21]: What do you think about putting the 3D model above the dimensioned drawing? It doesn't matter either way but I feel like it might look nicer.

File: docs/hw/mechanical.md
Comment (shamlian) [reply docs/hw/mechanical.md:24]: I agree, but not sure how to effect this on those other pages, since there's not a particular thing that is being footnoted.

**Review by shamlian — CHANGES_REQUESTED** [review]
This is on the whole a really great set of changes that helps the site look and work better. The only major concern I have is with the Total Power block on the electrical page.

File: docs/hw/mechanical.md
Comment (alsora) [reply docs/hw/mechanical.md:24]: Maybe we can assign the footnote to the first mention of "usb-c", "Bluetooth", "RaspberryPi", etc

File: docs/hw/mechanical.md
Comment (raphaelcherney) [reply docs/hw/mechanical.md:2]: Agree. I'll add some text there.

File: docs/hw/mechanical.md
Comment (raphaelcherney) [reply docs/hw/mechanical.md:17]: Yeah, I think this works pretty well actually. I made the change in commit b1041db if you want to check it out.

File: docs/hw/electrical.md
Comment (raphaelcherney) [reply docs/hw/electrical.md:34]: My thought was that a colored box would be more visible, but I agree with your point about the page structure and navigation. I changed it back to a section but kept the example as a call-out.

File: docs/hw/mechanical.md
Comment (raphaelcherney) [reply docs/hw/mechanical.md:14]: I moved it up and think it works quite well further up the page. Let me know if you disagree.

File: docs/hw/mechanical.md
Comment (raphaelcherney) [reply docs/hw/mechanical.md:2]: I moved them down the page some too. Let me know if you think they should be moved somewhere else.

File: docs/hw/electrical.md
Comment (shamlian) [inline docs/hw/electrical.md:32]: remove "to" (I will push this fix since it's trivial)

File: docs/hw/electrical.md
Comment (shamlian) [reply docs/hw/electrical.md:34]: Looks great; thank you!

File: docs/hw/mechanical.md
Comment (shamlian) [reply docs/hw/mechanical.md:2]: I think this is great.

**Review by shamlian — APPROVED** [review]
Approved; let's merge this after @alsora approves #13 to minimize conflicts.

**Review by shamlian — APPROVED** [review]

**Pull request merged**
